modules/200-operator-prometheus/images/prometheus-operator/patches/README.md

```diff
@@ -1,11 +0,0 @@
-# Patches
-
-## Scrape timestamp align
```

Comment:
Please, explain, whi have you deleted it?

Reply:
It was accepted in upstream, am i right?
And you added the flag below due to it was 10ms by default with our patch and now it isn't?

Reply:
Yes and no. The following [issue](https://github.com/prometheus-operator/prometheus-operator/issues/4851) has been implemented and now it's possible to specify any argument via `additionalArgs` in CR instead of hard-coding.

I've placed this info on top the issue description. Moved it from the `What is the expected result?` section to the `Description` section.